hack/update-gofmt.sh

```diff
@@ -36,6 +36,7 @@ find_files() {
         -o -wholename './release' \
         -o -wholename './target' \
         -o -wholename '*/third_party/*' \
+        -o -wholename '*/_vendor/*' \
```

Comment:
perhaps we should dodge all of staging instead?  @caesarxuchao staging is not hand-edited, right?